src/utils/streamParse.ts

```diff
@@ -4,6 +4,8 @@ import { AxiosResponse } from 'axios'
  * When running a /api/chat stream, the output needs to be parsed into an array of objects
  *  This method is used for development purposes and testing
  * 
+ * This will not work as intended with the inclusion of ollama-js, needs to be modified to work with it
+ * 
```

Comment:
progress made on planning for this issue??

Reply:
Uhhhh this is just an issue found in #19 

Not sure if I explained it enough, but I put a note here in case someone stumbles upon it.